add new overview topic (#2821)
influxdata/docs-v2: content/influxdb/cloud/notebooks/create-notebook.md

title: Create a notebook
description: >
  Create a notebook to explore, visualize, and process your data.
weight: 102
influxdb/cloud/tags:
menu:
  influxdb_cloud:
    name: Create a notebook
    parent: Notebooks
---

> For more information about cell types, see [Overview of Notebooks](/influxdb/cloud/notebooks/overview/). 

Reviewer: ```suggestion
To learn how notebooks can be useful and to see an overview of different cell types available in notebooks, see [Overview of Notebooks](/influxdb/cloud/notebooks/overview/).
```

1. In the navigation menu on the left, click **Notebooks**.

    {{< nav-icon "notebooks" >}}

> 2. Click **+Create Notebook**, then enter a name for your notebook in the **Name this notebook** field. By default, the notebook name appears as `<username>-notebook-<year>-<month>-<day and time>`.

Reviewer: ```suggestion
2. Click **+Create Notebook**, and then enter a name for your notebook in the **Name this notebook** field. By default, the notebook name appears as `<username>-notebook-<year>-<month>-<day and time>`.
```

3. Do the following at the top of the page:
   - Select your local time zone or UTC.
   - Choose a time range for your data.
4. Consider which notebook cells to add to your notebook. By default, each notebook includes **Metric Selector**, **Data Validation**, and **Visualization** cells. 

> 5. Click the **+** icon, and then add or more of the following cell types to your notebook:

Reviewer: ```suggestion
5. Click the **+** icon, and then add one or more of the following cell types to your notebook:
```

    - **Input cells**: to pull information into your notebook
    - **Transform cells**: to filter and apply changes to your data

>     - **Pass-through**: to visualize and add notes to your data

Reviewer: ```suggestion
    - **Pass-through cells**: to visualize and add notes to your data
```

>     - **Output**: to write data to a bucket, preview output before writing, or schedule an output task

Reviewer: ```suggestion
    - **Output cells**: to write data to a bucket, preview output before writing, or schedule an output task
```

    - Click **Preview** (or press **CTRL + Enter**) to preview the results of each cell in a raw data table without writing any data. 
    - Select **Run** to show the results of each cell and write it to the specified output bucket.

{{% /tab-content %}}
<!--------------------------------- END Input --------------------------------->

<!-------------------------------- BEGIN Transform-------------------------------->
{{% tab-content %}}

1. Select one of the following transform cell-types:
    - **Flux Script**: Use `__PREVIOUS_RESULT__` to build from data in the previous cell, enter a Flux script to transform your data. 
    - **Downsample**: Window data by time and apply an aggregate to each window to downsample data. For more information, see [Downsample data with notebooks](/influxdb/cloud/notebooks/downsample/).
2. Select **Preview** or **Run** in the upper left dropdown menu. By default, Preview appears.
    - Click **Preview** (or press **CTRL + Enter**) to preview the results of each cell in a raw data table without writing any data. 
    - Select **Run** to show the results of each cell and write it to the specified output bucket.

   {{% warn %}}

> If your cell contains a custom script that uses any output function to write data to InfluxDB (the `to()` function) or send it to a 3rd party service, clicking **Preview** will write data.

Reviewer: @mluu12 looks like this warning has a couple formatting issues with indent, space not around Preview with the bold, and to() is below the main line. Let's drop the backticks around the function and bold for now & indent text

    - Click **Preview** (or press **CTRL + Enter**) to preview the results of each cell in a raw data table without writing any data. 
    - Select **Run** to show the results of each cell and write it to the specified output bucket.

{{% warn %}}
If your cell contains a custom script that uses any output function to write data to InfluxDB (the to() function) or send it to a 3rd party service, clicking Preview will write data.
{{% /warn %}}

### Pass-through 

Select one of the following pass-through cell-types: 

- To change visability and name of columns, select **Column Editor**. 
- To create a visualization of your data, select **Visualization**. For details on available visualization types and how to use them, see [Visualization types](/influxdb/cloud/visualize-data/visualization-types/). 
- To enter explanatory notes, select **Markdown**. 

   - **Column Editor**:  
     - Click the toggle to hide or view your column. 
     - Rename your columns by hovering over the column name and pressing the pencil icon. 
   - **Visualization**: 
     - Change your graphic type by clicking the second dropdown menu at the top of the cell. By default, ig is a graph. 

>      - Choose specific values in your histogram graphic by clicking the dropdown menu labled **Select**. 

Reviewer: ```suggestion
     - For histograms, specify values by clicking **Select**. 
```

>      - Press the nut icon to configure graphic visualization. 

Reviewer: ```suggestion
     - To configure the visualization, click **Configure**. 
```

     - Click the **CSV** button to download results as an annotated CSV file. 
     - Export to the dashboard by clicking the labled button. 
   - **Markdown**: 

>      - Select **Edit** to type in your notes.  
>      - Select **Preview** to view your notes. 

Reviewer: ```suggestion
     - Type in notes as needed.
```

### Output 

Select one of the following output cell-types:

> - To recieve an alert, select **Notification** and complete the steps below. 

Reviewer: ```suggestion
- To recieve an alert, select [Notification](#add-a-notification) and complete the steps below. 
```

- To write output to a bucket, select **Output to Bucket** and complete the steps below. 
- To create a task, select **Schedule** and complete the steps below. 

> **Notification**

Reviewer: ```suggestion
#### Add a notification
```

1. Enter a time range to automatically check the data and enter your query offset. 

> 2. Customize the conditions to send an alarm. 

Reviewer: ```suggestion
2. Customize the conditions to send an alert. 
```

> 3. Choose one of the following to recieve your alarms: 

Reviewer: ```suggestion
3. Select an endpoint to receive an alert: 
```

   - Slack and a Slack Channel 
   - HTTP post 
   - Pager Duty

> 4. (Optional) Personalize your alarm message. By default, the message is "Notification Rule: ${ r._notification_rule_name } triggered by check: ${ r._check_name }: ${ r._message }." 

Reviewer: ```suggestion
4. (Optional) Personalize your message. By default, the message is "Notification Rule: ${ r._notification_rule_name } triggered by check: ${ r._check_name }: ${ r._message }." 
```

5. Click **Export as Task** to create your alarm. 

> **Output to Bucket**

Reviewer: ```suggestion
**Output to bucket**
```
